tensorflow/compiler/xla/tools/replay_computation.cc

```diff
@@ -234,7 +234,13 @@ StatusOr<Literal> ReplayComputation(const HloSnapshot& module,
     // Run fake computations with debug options ignoring XLA_FLAGS.  Users very
     // likely want XLA_FLAGS only to apply to the "real" computation being run,
     // not to the fake computations we use for generating arguments.
```

Comment:
`This comment needs to be updated to incorporate this new behavior.` comments from internal review , can you please update

Reply:
I just updated it. I merged both comment to make it more clear.